src/device-registry/models/Event.js

```diff
@@ -64,6 +64,24 @@ const valueSchema = new Schema({
     uncertaintyValue: { type: Number, default: null },
     standardDeviationValue: { type: Number, default: null },
   },
+  s1_pm1: {
+    value: {
```

Comment:
introducing the new Schema field, with plans to leave the pm2_5 for average measurements only. 

Just wondering @NoahNsimbe , could we adopt this across our entire pipeline? Where we introduce `s1_pm2_5` values and leave the `pm2_5 `to store the "average" values only? Same suggestion applies for pm1 and pm10.